.github/workflows/ci.yaml

```diff
@@ -86,35 +88,35 @@ jobs:
           goarch: ${{ matrix.goarch }}
           extra_files: CHANGELOG.md LICENSE README.md
 
-  release-darwin-arm64:
-    name: Build binaries for darwin arm64
-    runs-on: macos-14
-    needs: [release]
+  # release-darwin-arm64:
```

Comment:
Não seria melhor remover ?

Reply:
For sure, we can remove. This is a leftover from another CI.